compiler/include/LoopStmt.h

```diff
@@ -67,6 +73,7 @@ class LoopStmt : public BlockStmt
   bool                   mOrderIndependent;
   bool                   mVectorizationHazard;
   bool                   mParallelAccessVectorizationHazard;
+  LLVMMetadataList      mLLVMMetadataList;
```

Comment:
I am not a big fan of these files in our compiler where there's ton of spaces because it makes things unmaintainable in my view. As an example: could you do something here to make things aligned? :) I wouldn't mind if you removed all the spaces yourself, but others might.